### DISCUSSION TITLE
Running on localhost with the docker setup, which port/url to use?

### DISCUSSION THREAD

**Discussion by notflip** [dyrector-io/dyrectorio#902] [Q&A]
I just cloned the repo, copied the `.env.example` to `.env` and ran `docker compose up -d`.

Now all seems to be working fine! But I don't know how to access the UI?

**Comment by polaroi8d** [comment]
Hi @notflip,

If you haven't changed the configs, you should check the UI on `localhost:8000`, but you can easily configure this via the `.env` files. If you can't reach your UI, you should check if your containers are running properly with the `docker ps` command. For more information about the setup process, you can find it on our [docs](https://docs.dyrector.io/) site, but I'm happy to help here too.

**Reply by notflip** [reply]
I just ran the default setup again, just copying .env.example to .env, started with docker compose up and there is nothing to see on localhost:8000 (or localhost:3000), because in the docker logs I can see some activity on port 3000.

**Reply by polaroi8d** [reply]
Okay, could you please show your Docker logs? Nothing appeared on the mentioned ports, am I right?

**Comment by polaroi8d** [comment]
Closing due the inactivity.
